### PR TITLE
Fix for FM-166

### PR DESCRIPTION
**Pull request by Tharakamora**
Added generateOpenMRSObsWithEncounter method to use in DiagnosticReortHandler

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/fhir/api/util/FHIRObsUtil.java [hunk 1]
```diff
@@ -379,6 +380,13 @@ public static Obs generateOpenMRSObs(Observation observation, List<String> error
 		}
 		return obs;
 	}
+
```
Comment (harsha89) [inline api/src/main/java/org/openmrs/module/fhir/api/util/FHIRObsUtil.java]: Where is this method will use? if there is no usage the purpose will not be address

**Comment by dkayiwa** [comment]
@Tharakamora do you have an answer to the question above?

**Comment by dkayiwa** [comment]
@Tharakamora are you still working on this?

**Comment by dkayiwa** [comment]
Closing because this method has no use. @Tharakamora feel free to object. :)

**Pull request closed without merging**